hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/container/placement/TestContainerPlacement.java

```diff
@@ -85,8 +85,8 @@ public void testCapacityPlacementYieldsBetterDataDistribution() throws
         null, true, null);
 
     for (int x = 0; x < opsCount; x++) {
-      long containerSize = random.nextInt(100) * OzoneConsts.GB;
-      long metadataSize = random.nextInt(100) * OzoneConsts.GB;
+      long containerSize = random.nextInt(10) * OzoneConsts.GB;
```

Comment:
Just curious, why do we need this change?

Reply:
`random.nextInt(100)` creates a `SCMException` as there is no space to place the container. Having a smaller container size doesn't cause this `SCMException`.